src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/attributes.html.php

```diff
@@ -1 +1 @@
-<?php echo $view['form']->block($form, 'widget_attributes') ?>
+<?php echo $view['form']->block($form, 'widget_attributes');
```

Comment:
I have same dilemmas therefore I have separated template changes into second commit.

When a template have `T_INLINE_HTML` we know it is a template and don't do changes like that. But here - we don't know that.  Big PHP chunk may be in non-template file too, and how big is big enough...

You are against removing `?>` in templates, are you also against adding missing `;` or new line for control structure block?

What about leaving only first commit ?